lib/services/livesync/livesync-service.ts

```diff
@@ -658,16 +659,19 @@ export class LiveSyncService extends EventEmitter implements IDebugLiveSyncServi
 
 			const watcher = choki.watch(patterns, watcherOptions)
 				.on("all", async (event: string, filePath: string) => {
+
 					clearTimeout(timeoutTimer);
 
 					filePath = path.join(liveSyncData.projectDir, filePath);
 
 					this.$logger.trace(`Chokidar raised event ${event} for ${filePath}.`);
 
-					if (event === "add" || event === "addDir" || event === "change" /* <--- what to do when change event is raised ? */) {
-						filesToSync.push(filePath);
-					} else if (event === "unlink" || event === "unlinkDir") {
-						filesToRemove.push(filePath);
+					if (!isAllowedFinalFile(filePath)) {
```

Comment:
Can I place `.aar` file in App_Resources? In case yes, what should happen here - we'll not include the files in filesToSync and... we should rebuild the application? Will this happen?

Reply:
We haven't talked about placing aar files in `App_Resources` so I wouldn't know what would happen.

Reply:
@rosen-vladimirov App_Resources/Android/libs (where aars can be placed) is watched separately and should force a rebuild

Reply:
@rosen-vladimirov I checked it out and when you have an `example.aar` file in the `App_Resources` and the `tns run android`  or `tns run android --syncAllFiles` is running and the `example.aar` is changed, the changes are being ignored. 
When the command is stopped and ran again, the `example.aar` forces a rebuild.